### DISCUSSION TITLE
Dorling cartogram with overlapping bubbles

### DISCUSSION THREAD

**Discussion by hski-github** [neocarto/bertin#70] [Q&A] [answered]
Thank you for creating bertin.js. It is really great to see this library! I makes map charts very powerful. :-)

One thing I noticed looking at Dorling Diagram at https://observablehq.com/@neocartocnrs/bertin-js-dorling-cartogram?collection=@neocartocnrs/bertin that there are some overlaps in bubbles in the middle of the image (setting radius to higher values). 

I thought this would be the original idea of Dorling vs. standard prop symbol?

<img width="332" alt="grafik" src="https://user-images.githubusercontent.com/1921015/196036205-9577f026-f70c-4f3b-a294-a935efc35add.png">

**Comment by neocarto** [comment, marked as answer]
To avaid overlap, you can add the **iteration** parameter with a value > 200. It's more accurate, but the calculation time is longer.

**Reply by hski-github** [reply]
Are you using D3 force layout with forceCollide?

**Comment by neocarto** [comment]
Yes
